src/test/java/core/basesyntax/service/StorageServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageDao;
+import core.basesyntax.db.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageServiceTest {
+    private static StorageService storageService;
+    private static StorageDao storageDao;
```

Comment:
I think it's better to use Storage directly (what if StorageDao does not work correctly?)